building/tooling/test-runners/interface.md

```diff
@@ -119,15 +119,15 @@ The following per-test statuses are valid:
 
 - `pass`: The test passed
 - `fail`: The test failed
-- `error`: The test errored
+- `error`: The test errored - that is, it did not return a value
```

Comment:
I'm not happy with this, but I think we can do better than just "The test errored".

Reply:
Yeah this is good enough I feel